nearmint/ops/Dockerfile

```diff
@@ -43,8 +43,8 @@ ENV TMHOME=/srv/near
 
 RUN ["tendermint", "init" ]
 
-COPY nearmint/ops/tendermint-config.toml /srv/near/config/config.toml
 COPY nearmint/ops/run_nearmint /usr/local/bin/run_nearmint
+COPY nearmint/ops/tendermint-config.toml /tmp/config.toml
```

Comment:
Some OS clear /tmp on timer. Better keep persistent stuff somewhere else.

Reply:
It's not really persistent, it's just temporary before it's copied to persistent disk